include/rive/renderer.hpp

```diff
@@ -52,18 +53,29 @@ namespace rive
 		int height() const { return m_Height; }
 	};
 
-	class RenderPath : public CommandPath
-	{
-	public:
-		RenderPath* renderPath() override { return this; }
-		void addPath(CommandPath* path, const Mat2D& transform) override
-		{
-			addRenderPath(path->renderPath(), transform);
-		}
-
-		virtual void addRenderPath(RenderPath* path,
-		                           const Mat2D& transform) = 0;
-	};
+    class RenderPath : public RefCnt {};
+
+    enum class FillRule {
+        nonZero,
+        evenOdd
+    };
+
+    enum class PathVerb {
+        move,
+        line,
+        quad,
+        cubic,
+        close,
+    };
+
+    /*  Returns an immutable RenderPath.
+     *
+     *  ptCount is the logical number of points. Since "pts" is passed as
+     *  an array of floats, there will be 2*ptCount number of floats in the array.
+     */
+    extern rcp<RenderPath> makeRenderPath(const float pts[], int ptCount,
+                                          const uint8_t verbs[], int verbCount,
+                                          FillRule);
```

Comment:
Looks good! Feasible to support new verbs in the future too (if we ever care to). Btw, we don't ever supply quads right now (although for text we may want to). We should move it to a method on the renderer too no?

Reply:
Indded, can add more, and can remove kQuad if we decide we don't need it (but yes, likely we will for opentype fonts)